source/core/CEntryPoints.cpp

```diff
@@ -28,13 +28,13 @@ VIREO_EXPORT void* EggShell_Create(TypeManagerRef parent)
     return TypeManager::New(parent);
 }
 //------------------------------------------------------------
-VIREO_EXPORT NIError EggShell_REPL(TypeManagerRef tm, const Utf8Char* commands, Int32 length)
+VIREO_EXPORT NIError EggShell_REPL(TypeManagerRef tm, const Utf8Char* commands, Int32 length, Boolean IsDebugEnabled)
```

Comment:
[nit] New parameter should be isDebuggingEnabled

Reply:
Done